### PR TITLE
Improved RPM benchmarks

### PR DESCRIPTION
**Pull request by g4s8**
For #382 - using incremental batch udate for RPM repository,
added proper setup methods for each iteration to copy
bundle repository to memory storage instead of reading from disk.

### CODE REVIEW

File: src/main/java/com/artipie/rpm/benchmarks/RpmBench.java [hunk 1]
```diff
@@ -71,13 +79,52 @@ public void setup() {
...
         }
-        this.storage = new FileStorage(Paths.get(RpmBench.BENCH_DIR));
+        this.storage = new InMemoryStorage();
+        final Storage src = new FileStorage(Paths.get(RpmBench.BENCH_DIR));
+        RpmBench.sync(src, this.storage);
+    }
+
+    @Setup(Level.Iteration)
+    public void setupIter() {
+        this.storage.delete(new Key.From("/repodata"));
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/rpm/benchmarks/RpmBench.java:89]: @g4s8 keys with empty parts are not allowed, see `com/artipie/asto/Key.java:172`, please, list items by `repodata` key and remove every item

File: src/main/java/com/artipie/rpm/benchmarks/RpmBench.java [hunk 2]
```diff
@@ -71,13 +79,52 @@ public void setup() {
         if (RpmBench.BENCH_DIR == null) {
             throw new IllegalStateException("BENCH_DIR environment variable must be set");
         }
-        this.storage = new FileStorage(Paths.get(RpmBench.BENCH_DIR));
+        this.storage = new InMemoryStorage();
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/rpm/benchmarks/RpmBench.java:82]: @g4s8 as we want to measure memory consumption later, I'm not sure that using `InMemoryStorage` is a good idea

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@g4s8 thanks, check my comment, please

File: src/main/java/com/artipie/rpm/benchmarks/RpmBench.java (hunk 1 above)
Comment (g4s8) [reply src/main/java/com/artipie/rpm/benchmarks/RpmBench.java]: @olenagerasimova thanks, fixed

File: src/main/java/com/artipie/rpm/benchmarks/RpmBench.java (hunk 2 above)
Comment (g4s8) [reply src/main/java/com/artipie/rpm/benchmarks/RpmBench.java:83]: @olenagerasimova in any case we need to measure memory consumption by some other tool - not JMH, so we will need to write yet another test for that.

**Review by olenagerasimova — APPROVED** [review]
@g4s8 thanks

**Pull request merged**